modules/op-release-notes-1-19.adoc

```diff
@@ -0,0 +1,396 @@
+
+* With this update, tasks in the web console now feature `displayName` properties, providing clearer, more user-friendly names within the {pipelines-title} interface. This behavior replaces the use of technical task names and brings the `displayName` parameter functionality, previously found in TektonDashboard, to the web console.
+
+* With this update, {pac} extends GitOps workflow support in GitLab by enabling GitOps comments on pushed commits. This enhancement allows users to receive pipeline status updates directly on commit events, in addition to merge requests. Previously, this feature was only available for GitHub. # (Need to confirm if this is added in Google Docs)
+
+* With this update, {pipelines-title} includes support for optional environment variables in Task and Pipeline specifications. This enhancement allows users to define environment variables that are not required to be present, providing greater flexibility when referencing values from secrets or config maps.
+
+* With this update, the {pipelines-title} entrypoint image includes support for FIPS compliance. The image is now built with appropriate cryptographic dependencies and configurations to meet FIPS requirements, ensuring compatibility in environments with strict security standards.
+
+* With this update, FIPS compliance is enhanced by adding `disable_spire` and `disable_tls` build tags to the entrypoint command compilation process. This enhancement ensures cryptographic symbols are properly removed from the entrypoint binary to meet compliance requirements.
+
+* With this update, parameter naming conventions across the `StepActions` feature have been unified for consistency. The updated `git-clone` `StepActions` uses consistent casing with `cache-fetch` and `cache-upload` `StepActions`.
+
+* With this update, the `/ok-to-test` memory feature is disabled by default. This precaution helps mitigate the risk of malicious code execution within testing environments. # (Need to confirm if this is added in Google Docs)
+
+* With this update, support is provided for expanding dynamic variables from remote pipeline definitions. This enhancement improves pipeline composition capabilities, which enables additional flexible pipeline architectures.
+
+* With this update, the {tekton-results} Watcher retries reconciliation multiple times before removing the finalizer, as long as the `storedDeadline` duration has not been exceeded. This improves reliability by reducing the chances of missing `TaskRun` or `PipelineRun` storage.
+
+* With this update, {tekton-results} supports field-based filtering for `Get` and `List` requests on `TaskRuns` and `PipelineRun`. When filter fields are specified, only the requested data is returned, improving response times and reducing network load.
```

Comment:
I believe this duplicates the bullet point on line 90:
> `* With this update, the {tekton-results} API supports response filtering to minimize payload size and enhance network efficiency. This update enables users to define the specific fields included in API responses, which significantly benefits List operations by preventing the retrieval of entire objects, thus optimizing performance.`

Reply:
let's go with the one in line 90 then